gms/api/src/main/pegasus/com/linkedin/dataprocess/DataProcessKey.pdl

```diff
@@ -0,0 +1,32 @@
+namespace com.linkedin.dataprocess
+
+import com.linkedin.common.FabricType
+
+/**
+ * Key for process resource
+ */
+record DataProcessKey {
+
+  /**
+   * Standardized orchestration platform urn where process is running. It can be Azure Data Factory or Sqoop script
```

Comment:
There is no URN class defined for this though, it's string type. Is it intentional?